src/NumericalAlgorithms/Interpolation/Callbacks/FindApparentHorizon.hpp

```diff
@@ -95,7 +96,8 @@ struct FindApparentHorizon {
       ERROR("FindApparentHorizon: Found points that are not in any block");
     }
 
-    const auto& verbosity = db::get<::Tags::Verbosity>(*box);
+    const auto& verbosity =
+        db::get<logging::Tags::Verbosity<InterpolationTargetTag>>(*box);
```

Comment:
I'm not sure why you chose the group to be `InterpolationTargetTag`.  Shouldn't it be `OptionTags::ApparentHorizon<InterpolationTargetTag,Frame>`?

Reply:
This template parameter just "labels" the tag, it doesn't need to be an option group if the tag is not created from options. I thought the `InterpolationTargetTag` would be the most fitting "label" here, since it represents "AhA", "AhB" etc if I understand correctly. I think the apparent horizon code should adopt option groups, and once it does we can use the option group as the label instead. Does that make sense?

Reply:
Ah, ok, at first I thought you were changing how Verbosity is specified in an input file.